zerver/tests/test_import_export.py

```diff
@@ -1406,7 +1406,12 @@ def get_muted_users(r: Realm) -> set[tuple[str, str, str]]:
 
         @getter
         def get_user_group_names(r: Realm) -> set[str]:
-            return {group.named_user_group.name for group in UserGroup.objects.filter(realm=r)}
+            result = set()
+            for group in UserGroup.objects.filter(realm=r):
+                if hasattr(group, "named_user_group"):
+                    result.add(group.named_user_group.name)
+
+            return result
```

Comment:
The unnamed `group_creator` user group was part of this and giving an error because it didn't have `named_user_group` attribute

Reply:
I'm not too familiar with this set of tests, so just wanted to check one thing. `get_named_user_group_names` already exists and `get_user_membership` already exists. Then do we even need this piece of test or a separate tests for anonymous groups?